samples/servicecontrol/adapter-rabbitmq-different-topologies/SCTransportAdapter_1/Sales/Program.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using NServiceBus;
+
+class Program
+{
+    static void Main()
+    {
+        AsyncMain().GetAwaiter().GetResult();
+    }
+
+    static async Task AsyncMain()
+    {
+        Console.Title = "Samples.ServiceControl.RabbitMQAdapter.Sales";
+        const string letters = "ABCDEFGHIJKLMNOPQRSTUVXYZ";
+        var random = new Random();
+
+        #region SalesConfiguration
+
+        var endpointConfiguration = new EndpointConfiguration("Samples.ServiceControl.RabbitMQAdapter.Sales");
+
+        var transport = endpointConfiguration.UseTransport<RabbitMQTransport>();
+        transport.ConnectionString("host=localhost");
+        transport.UseDirectRoutingTopology();
```

Comment:
It makes sense to add `transport.DelayedDelivery().DisableTimeoutManager();` here to avoid creating the timeout manager satellite queues, which just add even more noise when looking at the broker for this sample.